.jshintrc

```diff
@@ -10,8 +10,20 @@
   "strict": true,
   "undef": true,
   "unused": true,
+  "expr": true,
   "globals": {
     "angular": false,
-    "prettyPrintOne": false
+    "prettyPrintOne": false,
+    // MOCHA
```

Comment:
github shows this in red, is this ok? Like doesn't comments usually start with `#` at such files?